x-pack/test/functional_execution_context/test_utils.ts

```diff
@@ -9,17 +9,26 @@ import Path from 'path';
 import { isEqualWith } from 'lodash';
 import type { Ecs, KibanaExecutionContext } from '@kbn/core/server';
 import type { RetryService } from '@kbn/ftr-common-functional-services';
+import { concatMap, defer, filter, firstValueFrom, ReplaySubject, scan, timeout } from 'rxjs';
 
 export const logFilePath = Path.resolve(__dirname, './kibana.log');
 export const ANY = Symbol('any');
 
+let logstream$: ReplaySubject<Ecs> | undefined;
+
+export function getExecutionContextFromLogRecord(record: Ecs | undefined): KibanaExecutionContext {
+  if (record?.log?.logger !== 'execution_context' || !record?.message) {
+    throw new Error(`The record is not an entry of execution context`);
+  }
+  return JSON.parse(record.message);
+}
+
 export function isExecutionContextLog(
-  record: string | undefined,
+  record: Ecs | undefined,
```

Comment:
it used to be called as `isExecutionContextLog(logEntry.message, {...})`.

I refactored it to receive the `logEntry` which is in Ecs format in these tests.